packages/client/components/Dashboard/DashSidebar.tsx

```diff
@@ -57,8 +54,78 @@ const Wrapper = styled('div')({
   flexDirection: 'column'
 })
 
+const OrgName = styled('div')({
+  paddingTop: 8,
+  paddingLeft: 8,
+  fontWeight: 600,
+  fontSize: 12,
+  lineHeight: '24px',
+  color: PALETTE.SLATE_500,
+  [makeMinWidthMediaQuery(Breakpoint.SIDEBAR_LEFT)]: {
+    paddingLeft: 16
+  }
+})
+
+interface Props {
+  isOpen: boolean
+  viewerRef: DashSidebar_viewer$key | null
+}
+
 const DashSidebar = (props: Props) => {
-  const {isOpen, viewer} = props
+  const {isOpen, viewerRef} = props
+  const location = useLocation()
+  const {pathname} = location
+
+  const viewer = useFragment(
+    graphql`
+      fragment DashSidebar_viewer on User {
+        ...StandardHub_viewer
+        ...DashNavList_viewer
+        featureFlags {
+          checkoutFlow
+        }
+        organizations {
+          id
+          name
+        }
+      }
+    `,
+    viewerRef
+  )
+
+  if (!viewer) return null
+  const {featureFlags, organizations} = viewer
+  const showOrgSidebar = featureFlags.checkoutFlow && pathname.startsWith(`/me/organizations`)
+
+  if (showOrgSidebar) {
+    const orgIdFromPathname = pathname.split('/')[3]
+    const currentOrg = organizations.find((org) => org.id === orgIdFromPathname)
+    const {id: orgId, name} = currentOrg ?? {}
+    return (
+      <Wrapper>
+        <SideBarStartMeetingButton isOpen={isOpen} />
+        <Nav isOpen={isOpen}>
+          <Contents>
+            <NavItemsWrap>
+              <NavItem icon={'arrowBack'} href={'/me'} label={'Back'} />
+              <OrgName>{name}</OrgName>
+              <NavItem
+                icon={'creditScore'}
+                href={`/me/organizations/${orgId}/billing`}
+                label={'Plans & Billing'}
+              />
+              <NavItem
+                icon={'group'}
+                href={`/me/organizations/${orgId}/member`}
```

Comment:
-1 The url should be 'members', otherwise it won't open member tab

Reply:
Thanks, updated 👍